legacy/include/distconv/cudnn/backend.hpp

```diff
@@ -1,611 +1,40 @@
 #pragma once
+#ifndef H2_LEGACY_INCLUDE_DISTCONV_CUDNN_BACKEND_HPP_INCLUDED
+#define H2_LEGACY_INCLUDE_DISTCONV_CUDNN_BACKEND_HPP_INCLUDED
```

Comment:
Shouldn't this file be renamed to something like `dnn_lib` to be consistent with LBANN and the multiplexing between backends?

Reply:
Eh, maybe? But it doesn't really matter. The namespace gets abstracted below.

Reply:
I understand that it get’s abstracted, but to have the top of the abstraction tree start “biased” to cudnn is a big confusion.